### ISSUE TITLE
Lingobit Localizer translation strings are not accessible with NVDA, it is non standard control of UI of The Lingobit Localizer

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3887]
**Reported by janusz on 2014-02-15 18:37**
Dear developers of NVDA excellent screen reader.
I have found out, that NVDA can not speak The string inside open project when user want to translate some supported files by using Lingobit Localizer. This tool is supporting translation of .html, .cpp sources and other modules, including Net framework binaries such as .dll and .exe. But unfortunately thanks to The non standard GUI of this program, user can not navigate inside The translation strings. It seems to me, that UI of this program is using Microsoft Trident rendering engine and there is also a non standard class name for object which is being used for displaying The translatable strings.
The direct download link of Lingobit Localizer  is

http://www.lingobit.com/localizer.exe
The non standard class name which is very probably causing issues is

MFCGridCtrl
Do You think, that You would try to add support for this class name? May be, that problem is caused by other issue. Please keep on mind, that this tool is perfect opportunity for blind translators, it is not my personal toy i would like to enable other visually impaired translators to use this tool in productive environment.
Thank You very much for Yours help and discussion.

**Comment by Adriani90** [comment]
@janusz are you still having this issue?

**Comment by Adriani90** [comment]
cc: @leonardder

**Comment by Adriani90** [comment]
I am testing with NVDA 2019.1.1 and Lingobit 9.0. The application seems to be quite inaccessible. Apart from some dialogs the interface is not really usable with NVDA. Is it possible to implement the class referenced in the description to make it somehow accessible? There are many companies and independent translators using this software professionally, especially in Germany.
cc: @feerrenrut, @michaelDCurran

**Comment by feerrenrut** [comment]
The bast pathway forward is to contact the developers of Lingobit, we would be happy to work with them to try to improve the accessibility of their software.

**Comment by Adriani90** [comment]
@feerrenrut they have a really bad support, actually even their contact form does not work and they update the software every 4 year or so, if at all.
So I think supporting this software via display model would be a good idea here because we do not expect many updates in short time intervals.

**Comment by Adriani90** [comment]
This software has not been updated since 2016, copyright seems to have ended in 2016 and the last version is still 9.0.
Closing as abandoned.
Please contact the company and create a new issue in case this is still relevant.
